contracts/verifier/RelayVerifier.sol

```diff
@@ -58,5 +58,6 @@ contract RelayVerifier is IRelayVerifier, Ownable {
     function acceptToken(address token) external onlyOwner {
         require(token != address(0), "Token cannot be zero address");
         tokens[token] = true;
+        emit AllowedToken(token);
```

Comment:
There is no need to add events to determine which tokens are accepted, the relay server can query the state of the contract directly and that is an off-chain call without any cost. This scales better than the use of events. For simplicity, the verifiers can have a list of accepted tokens (additionally to the current map) so parsing the state is easier.